### PR TITLE
Fix bugs in User Guide

### LINKED ISSUE #291: Non working examples in UG
Will need to check through UG to make sure all examples are working

Errors found:
1. add patient remark examples missing prefix and have spelling mistakes
image

### PR DESCRIPTION
**Pull request by yanghengtang**
Closes #291

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -813,17 +816,21 @@ Back to [Table of Contents](#table-of-contents).
 | **View Appointment**         | `view_a INDEX`<br> e.g., `view_a 3`                                                                                                                           |
 | **Edit Patient**             | `edit_p INDEX [ic\NRIC] [n\NAME] [p\PHONE]`<br> e.g.,`edit_p 3 ic\T0212385J n\Joe Ng`                                                                         |
 | **Edit Doctor**              | `edit_d INDEX [ic\NRIC] [n\NAME]`<br> e.g.,`edit_d 2 ic\T0212385J n\Joe Ng`                                                                                   |
+| **Edit Appointment**         | `edit_a INDEX [pic\PATIENT_NRIC] [dic\DOCTOR_NRIC] [from\START_TIME] [end\END_TIME]`<br> e.g.,`edit_a 2 pic\T0212385J`                                        |
```
Comment (derekjxtan) [inline docs/UserGuide.md:819]: `end\` change to `\to`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -614,14 +617,14 @@ Back to [Appointment Management Features](#appointment-management-features).
...
-**Format**: `edit_a INDEX [pic\PATIENT_NRIC] [dic\DOCTOR_NRIC] [from\START_TIME] [end\END_TIME]`
+**Format**: `edit_a INDEX [pic\PATIENT_NRIC] [dic\DOCTOR_NRIC] [from\START_TIME] [to\END_TIME]`
 
 * Edits the appointment at the specified `INDEX`. The index refers to the index number shown in the displayed appointment list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 
 **Examples**:
-*  `edit_a 1 pic\S9912343G dic\T0212385J from\2023-09-11 07:30 to\2023-09-11 08:00` Edits the patient and doctor NRIC of the 1st appointment to be `S9912343G` and `T0212385J` respectively and the start time and end time of the appointment to be `2023-09-11 07:30` and `2023-09-11 08:00` respectively
+*  `edit_a 1 pic\S9912343G dic\T0212385J from\2023-09-11 07:30 to\2023-09-11 08:00` edits the patient and doctor NRIC of the 1st appointment to be `S9912343G` and `T0212385J` respectively and the start time and end time of the appointment to be `2023-09-11 07:30` and `2023-09-11 08:00` respectively
```
Comment (mounilsankar) [inline docs/UserGuide.md:627]: Is there a reason why all the edit commands dont have "Entering" at the front?

Comment (yanghengtang) [reply docs/UserGuide.md:627]: more user centric

File: docs/UserGuide.md [hunk 3]
```diff
@@ -232,8 +235,8 @@ Allows you to view details, such as basic information, medical condition and rem
 * The index **must be a positive integer** 1, 2, 3, …​
 
 **Examples**:
-* `list_p` followed by `view_p 2` views the 2nd patient in the patient list.
-* `find_p Jonathan` followed by `view_p 1` views the 1st patient in the results of the `find_p` command.
+* Entering `list_p` followed by `view_p 2` views the 2nd patient in the patient list.
```
Comment (cheeggered) [inline docs/UserGuide.md:238]: instead of 'views' maybe 'shows the details of' would be more grammatically correct.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -251,8 +254,8 @@ Allows you to edit an existing patient in the system. This is useful to correct
 * Existing values will be updated to the input values.
 
 **Examples**:
-*  `edit_p 1 n\Joe Ng` Edits the name of 1st patient to be `Joe Ng`.
-*  `edit_p 2 p\91234567` Edits the phone of the 2nd patient to be `91234567`.
+*  `edit_p 1 n\Joe Ng` edits the name of 1st patient to be `Joe Ng`.
```
Comment (cheeggered) [inline docs/UserGuide.md:257]: '1st patient in the patient list' would be clearer, can follow the format in delete_p (line 311)

File: docs/UserGuide.md [hunk 5]
```diff
@@ -434,8 +437,8 @@ Allows you to view details, such as basic information, specialisation and remark
 * The index **must be a positive integer** 1, 2, 3, …​
 
 **Examples**:
-* `list_d` followed by `view_d 2` views the 2nd doctor in the doctor list.
-* `find_d Jonathan` followed by `view_d 1` views the 1st doctor in the results of the `find_d` command.
+* Entering `list_d` followed by `view_d 2` views the 2nd doctor in the doctor list.
+* Entering `find_d Jonathan` followed by `view_d 1` views the 1st doctor in the results of the `find_d` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:441]: same as above, 'shows the details of the' instead of 'views'

File: docs/UserGuide.md [hunk 6]
```diff
@@ -454,7 +457,7 @@ Allows you to edit an existing doctor in the system. This is useful to correct a
 * Existing values will be updated to the input values.
 
 **Examples**:
-*  `edit_d 1 n\Joe Ng ic\T0212385J` Edits the name and NRIC of the 1st doctor to be `Joe Ng` and `T0212385J` respectively.
+*  `edit_d 1 n\Joe Ng ic\T0212385J` edits the name and NRIC of the 1st doctor to be `Joe Ng` and `T0212385J` respectively.
```
Comment (cheeggered) [inline docs/UserGuide.md:460]: same as edit_p, '1st doctor in the doctor list' would be clearer

File: docs/UserGuide.md [hunk 7]
```diff
@@ -507,8 +510,8 @@ Allows you to delete a specific doctor from the system.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 **Examples**:
-* `list_d` followed by `delete 2` deletes the 2nd doctor in the doctor list.
-* `find_d Jonathan` followed by `delete 1` deletes the 1st doctor in the results of the `find_d` command.
+* Entering `list_d` followed by `delete_d 2` deletes the 2nd doctor in the doctor list.
+* Entering `find_d Jonathan` followed by `delete_d 1` deletes the 1st doctor in the results of the `find_d` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:514]: 'results' can be a bit vague, can be 'resultant doctor list' instead

File: docs/UserGuide.md [hunk 8]
```diff
@@ -528,8 +531,8 @@ Allows you to add a specialisation for a specified doctor in the system.
 * Only one specialisation can be added at a time.
 
 **Examples**:
-* `list_d` followed by `add_tag_d 2 Orthopaedic` add specialisation the 2nd doctor in the doctor list.
-* `find_d Jonathan` followed by `add_tag_d 1 Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
+* Entering `list_d` followed by `add_tag_d 2 t\Orthopaedic` add specialisation the 2nd doctor in the doctor list.
+* Entering `find_d Jonathan` followed by `add_tag_d 1 t\Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:535]: same as above, can use 'resultant doctor list' instead of 'results'

File: docs/UserGuide.md [hunk 9]
```diff
@@ -546,24 +549,24 @@ Allows you to delete a specialisation from a specific doctor in the system.
 * Only one specialisation can be deleted at a time.
 
 **Examples**:
-* `list_d` followed by `delete_tag_d 2 Orthopaedic` add specialisation the 2nd doctor in the doctor list.
-* `find_d Jonathan` followed by `delete_tag_d 1 Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
+* Entering `list_d` followed by `delete_tag_d 2 t\Orthopaedic` add specialisation the 2nd doctor in the doctor list.
+* Entering `find_d Jonathan` followed by `delete_tag_d 1 t\Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:553]: same here

File: docs/UserGuide.md [hunk 10]
```diff
@@ -546,24 +549,24 @@ Allows you to delete a specialisation from a specific doctor in the system.
...
 
 * Edits the remark of doctor at the specified `INDEX`.
 * The index refers to the index number shown in the displayed doctor list.
 * The index **must be a positive integer** 1, 2, 3, …
 
 **Examples**:
-* `list_d` followed by `remark_d 2 Doctor will not be in clinic until 30/12/2023` add specialisation the 2nd doctor in the doctor list.
-* `find_d Jonathan` followed by `remark_d 1 Doctor will not be in clinic until 30/12/2023` add specialisation the 1st doctor in the results of the `find_d` command.
+* Entering `list_d` followed by `remark_d 2 r\Doctor will not be in clinic until 30/12/2023` add specialisation the 2nd doctor in the doctor list.
+* Entering `find_d Jonathan` followed by `remark_d 1 r\Doctor will not be in clinic until 30/12/2023` add specialisation the 1st doctor in the results of the `find_d` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:569]: same here

File: docs/UserGuide.md (hunk 2 above)
Comment (cheeggered) [inline docs/UserGuide.md:627]: '1st appointment' can be '1st appointment in the appointment list'

File: docs/UserGuide.md [hunk 11]
```diff
@@ -651,7 +654,7 @@ Allows you to view details, such as basic information, prescription given and re
 * The index **must be a positive integer** 1, 2, 3, …​
 
 **Examples**:
-* `list_a` followed by `view_a 2` views the 2nd appointment in the appointment list.
+* Entering `list_a` followed by `view_a 2` views the 2nd appointment in the appointment list.
```
Comment (cheeggered) [inline docs/UserGuide.md:657]: same as view_p and view_d, can use 'shows the details of' instead of 'views'

File: docs/UserGuide.md [hunk 12]
```diff
@@ -676,24 +679,24 @@ Back to [Appointment Management Features](#appointment-management-features).
...
 * Adds a prescription to the appointment at the specified `INDEX`.
 * The index refers to the index number shown in the displayed appointment list.
 * The index **must be a positive integer** 1, 2, 3, …
 * Only one prescription can be added at a time.
 
 **Examples**:
-* `list_a` followed by `add_tag_a 2 Panadol` adds the prescription `Panadol` to the 2nd appointment in the appointment list.
-* `list_a pic\S9923334G` followed by `add_tag_a 1 Panadol` adds the prescription to the 1st appointment in the result of the `list_a` command.
+* Entering `list_a` followed by `add_tag_a 2 t\Panadol` adds the prescription `Panadol` to the 2nd appointment in the appointment list.
+* Entering `list_a pic\S9923334G` followed by `add_tag_a 1 t\Panadol` adds the prescription to the 1st appointment in the result of the `list_a` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:691]: 'result' to 'resultant appointment list'

File: docs/UserGuide.md [hunk 13]
```diff
@@ -702,16 +705,16 @@ Allows you to delete a prescription from an appointment in the system.
 * Only one prescription can be deleted at a time.
 
 **Examples**:
-* `list_a` followed by `delete_tag_a 2 Panadol` deletes the prescription `Panadol` from the 2nd appointment in the appointment list.
-* `list_a pic\S9923334G` followed by `delete_tag_a 1 Panadol` deletes the prescription `Panadol` form the 1st appointment in the results of the `list_a` command.
+* Entering `list_a` followed by `delete_tag_a 2 t\Panadol` deletes the prescription `Panadol` from the 2nd appointment in the appointment list.
+* Entering `list_a pic\S9923334G` followed by `delete_tag_a 1 t\Panadol` deletes the prescription `Panadol` form the 1st appointment in the results of the `list_a` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:709]: 1. typo, 'form' should be 'from'
2. 'results' to 'resultant appointment list'

File: docs/UserGuide.md [hunk 14]
```diff
@@ -723,8 +726,8 @@ Note that if the `REMARK` is empty, the existing remarks of the appointment will
 </div>
 
 **Examples**:
-* `list_a` followed by `remark_a 2 Patient does not need any follow up` adds the remark `Patient does not need any follow up` to the 2nd appointment in the appointment list.
-* `list_a pic\S9923334G` followed by `remark_a 1 Patient does not need nay follow up` adds the remark `Patient does not need any follow up` to the 2nd appointment in the results of the `list_a` command.
+* Entering `list_a` followed by `remark_a 2 r\Patient does not need any follow up` adds the remark `Patient does not need any follow up` to the 2nd appointment in the appointment list.
+* Entering `list_a pic\S9923334G` followed by `remark_a 1 r\Patient does not need nay follow up` adds the remark `Patient does not need any follow up` to the 2nd appointment in the results of the `list_a` command.
```
Comment (cheeggered) [inline docs/UserGuide.md:730]: change 'results' as above

File: docs/UserGuide.md [hunk 15]
```diff
@@ -345,16 +348,16 @@ Allows you to delete a medical condition from a specified patient in the system.
...
+* Entering `find_p Jonathan` followed by `delete_tag_p 1 t\cancer` deletes the medical condition `cancer` of the 1st patient in the results of the `find_p` command.
 
 Back to [Patient Management Features](#patient-management-features).
 
 ### Editing the remark of a patient : `remark_p`
 
 Allows you to edit the remark of a specified patient in the system. The patient's remark can be used to store temporary information such as family health history and current medications/treatments patients are on.
 
-**Format**: `remark_p INDEX [r\REMARK]`
+**Format**: `remark_p INDEX r\REMARK`
```
Comment (cheeggered) [inline docs/UserGuide.md:360]: remark should be optional, when they don't input a remark, mediconnect will delete the pre-existing remark stored in the specified patient/appointment/doctor.

eg. remark_p 1 works, and returns this
image

File: docs/UserGuide.md (hunk 4 above)
Comment (yanghengtang) [reply docs/UserGuide.md:257]: speaking of which, why never add when it is your turn to handle this entire section?

**Review by derekjxtan — APPROVED** [review]
lgtm

File: docs/UserGuide.md (hunk 4 above)
Comment (cheeggered) [reply docs/UserGuide.md]: so soz missed it out haha just realised

**Pull request merged**